packages/cli/commands/sandbox/create.js

```diff
@@ -37,18 +120,47 @@ exports.handler = async options => {
       name: sandboxName,
     })
   );
+  let result;
+  try {
+    result = await createSandbox(accountId, sandboxName).then(
+      ({ name, sandboxHubId }) => {
+        logger.success(
+          i18n(`${i18nKey}.success.create`, {
+            name,
+            sandboxHubId,
+          }),
+          `
 
-  return createSandbox(accountId, sandboxName).then(
-    ({ name, sandboxHubId }) => {
-      logger.success(
-        i18n(`${i18nKey}.success.create`, {
-          name,
-          sandboxHubId,
-        })
+The following step will prompt you to authenticate with sandbox "${name}" and add it to the CLI config.
+          `
+        );
+        return { name, sandboxHubId };
+      }
+    );
+  } catch (err) {
+    if (
+      err.error &&
+      err.error.category &&
+      err.error.category === 'MISSING_SCOPES'
+    ) {
+      const websiteOrigin = getHubSpotWebsiteOrigin(env);
+      const url = `${websiteOrigin}/personal-access-key/${accountId}`;
+      logger.info(
```

Comment:
We may want to get some design eyes on this message.

Reply:
what's the message? Forgive my Github incompetence.